packages/sdk-middleware-logger/src/logger.js

```diff
@@ -6,13 +6,19 @@ import type {
   Next,
 } from 'types/sdk'
 
-export default function createLoggerMiddleware(): Middleware {
+export default function createLoggerMiddleware(
+  cb: (request: MiddlewareRequest, response: MiddlewareResponse) => void
+): Middleware {
   return (next: Next): Next => (
     request: MiddlewareRequest,
     response: MiddlewareResponse
   ) => {
     const { error, body, statusCode } = response
     /* eslint-disable */
+    // Pass function to customize logging.
+    if (cb) {
+      cb(request, response)
+    }
```

Comment:
Hmm, this should work but I'm wondering why the customer wouldn't just implement their own logging middleware instead?

Reply:
I think it's still fine to allow custom logging in this middleware, but then I would also create a default callback function which logs to the console